packages/core/test/Proxy.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from 'chai';
+import { Contract, Signer } from 'ethers';
+import { ethers, upgrades } from 'hardhat';
+import { HMToken, Staking } from '../typechain-types';
+
+let escrowFactoryProxyContract: Contract,
+  HMTokenContract: HMToken,
+  stakingContract: Staking,
+  adminImplementation: Contract;
+let owner: Signer, account1: Signer;
+
+describe('Proxy', function () {
```

Comment:
I'd make this test part of EscrowFactory. We'll have Proxy for RewardPool & Staking, as well, so it's not appropriate to call this test `Proxy`.

Reply:
I've changed it to a more specific name to avoid problems when you add the others